moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -1130,6 +1139,8 @@ bool ConfigurationFilesWidget::generatePackage()
     }
   }
 
+  loadTemplateStrings();
+
```

Comment:
They might be cleared here:
https://github.com/ros-planning/moveit/blob/431fa5b159a860ada99192b800ab75df6c6b306b/moveit_setup_assistant/src/widgets/configuration_files_widget.cpp#L1252-L1253
```
  // Clear template strings in case this function is run multiple times with changes in between
  template_strings_.clear();
```